rules/firestore.json

```diff
@@ -1,15 +1,52 @@
 rules_version = "2";
 service cloud.firestore {
     match /databases/{database}/documents {
+        match /{document=**} {
+            // by default, nothing can be read or written unless explicitly authorized
+            allow read, write: if false;
+        }
         match /foo/{document=**} {
-            // only used for testing purposes, do not push to production 
+            // only used for testing purposes, do not push to production
```

Comment:
isn't it like, about to be pushed in production :p ?

Reply:
nope, ce document est pas encore pushé vu qu'on a qu'un seul firebase, auquel cas on risquerait d'avoir des soucis entre master et dev. Donc faut le pusher manuellement qu'après enmasterisation, et enlever cela. Mais à terme, ce /foo/ disparaitra, d'ici la fin du gros ticket sécu.

Reply:
Quand ce code atteindra la branche master ce code là sera pas en application sur master ?

Reply:
non, ce fichier est à copier/coller dans la console firebase manuellement pour le moment